data/assets/examples/renderable/renderablemodel/model.asset

```diff
@@ -0,0 +1,52 @@
+-- Basic
```

Comment:
How about moving this into a `model_lighting.asset` and then have another basic one without lighting?  Since that would be a bit simpler still